scim/api.py

```diff
@@ -58,3 +64,49 @@ def create_user(request, scim_user: CreateUserRequest) -> tuple[int, User | dict
         return 201, user
     except UserExists:
         return 409, {"message": "A user with that ID already exists"}
+
+
+@router.put(
+    "scim/v2/Users/{id}",
+    response={200: UpdateUserResponse, 201: CreateUserResponse, 409: Error},
+)
+def update_user_and_profile(
```

Comment:
i see why you renamed this but rather use "identity" i think? Really, in terms of SCIM I think user is still kinda the right name :D 

```suggestion
def update_identity(
```